source/features/locked-issue-indicator.css

```
@@ -0,0 +1,3 @@
+:has(reactions-menu) .rgh-locked-issue-indicator {
```

Let's use the same selector 
```suggestion
:has(.js-pick-reaction:first-child) .rgh-locked-issue-indicator {
```

---

The selector is intentional, it needs to re-hide the locked label caused by https://github.com/refined-github/refined-github/pull/7051#issuecomment-1809450140. It should only activate when the reaction button does exist, not when it doesn't.

---

Oh right, so it should be 
```suggestion
:has(.js-pick-reaction:not(:first-child)) .rgh-locked-issue-indicator {
```

The point is that we should try to use the same exact logic or else it might get out of sync. If GitHub changes the element name, the feature works, but not this piece, and we'll never notice.